### PR TITLE
feature(surveys): support multiple surveys

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: .github/workflows/prod-build.yml [hunk 1]
```diff
@@ -176,9 +176,12 @@ jobs:
           REACT_APP_ENABLE_PLUS_EU: true
 
           # Document survey (https://github.com/mdn/yari/pull/6371).
-          REACT_APP_SURVEY_START: 0 # (new Date("2022-06-..")).getTime()
-          REACT_APP_SURVEY_END: 0 # (new Date("2022-06-..")).getTime()
-          REACT_APP_SURVEY_RATE: 0.05 # 5 Percent.
+          REACT_APP_SURVEY_START_CSS_CASCADE_2022: 1655208000000 # (new Date("2022-06-14 12:00:00Z")).getTime()
```
Comment (schalkneethling) [inline .github/workflows/prod-build.yml:179]:
```suggestion
          REACT_APP_SURVEY_START_CSS_CASCADE_2022: 1655208000000 # (new Date("2022-06-14 12:00:00Z").getTime()
```

The timestamp is all good though :)

File: .github/workflows/prod-build.yml [hunk 2]
```diff
@@ -176,9 +176,12 @@ jobs:
...
+          REACT_APP_SURVEY_END_CSS_CASCADE_2022: 1655812800000 # (new Date("2022-06-21 12:00:00Z")).getTime()
```
Comment (schalkneethling) [inline .github/workflows/prod-build.yml:180]:
```suggestion
          REACT_APP_SURVEY_END_CSS_CASCADE_2022: 1655812800000 # (new Date("2022-06-21 12:00:00Z").getTime())
```

File: .github/workflows/prod-build.yml (hunk 1 above)
Comment (fiji-flo) [reply .github/workflows/prod-build.yml:179]: Those parenthesis come from Claas and I think they where intentional like that. Doesn't make a difference :)

**Review by schalkneethling — APPROVED** [review]
LGTM, thanks @fiji-flo 🏄

**Pull request merged**